### PR TITLE
RET 362

### PR DESCRIPTION
**Pull request by brendanmchugh**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RET-362

### Change description ###

Add the video hearings question page

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/views/video-hearings.njk [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  {{ govukBackLink({
+  text: "Back",
+  href: "/"
+}) }}
+{% endblock %}
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+
+      {% if noRadioButtonSelectedError %}
```
Comment (marcjudgey) [inline src/main/views/video-hearings.njk:18]: Small thing, could you please line up the start and the end of the if statement

File: src/main/controllers/VideoHearingsController.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    else if (req.body['video-hearing'] === 'no' && req.body.saveButton === 'saveContinue') {
+      res.redirect('/steps-to-making-your-claim');
+    }
+    else if (req.body['video-hearing'] === 'yes' && req.body.saveButton === 'saveForLater') { 
+      res.redirect('/your-claim-has-been-saved');
+    }
+    else if (req.body['video-hearing'] === 'no' && req.body.saveButton === 'saveForLater') { 
+      res.redirect('/your-claim-has-been-saved');
+    }
```
Comment (tautvydas-cerniauskas) [inline src/main/controllers/VideoHearingsController.ts:23]: This logic seems really overly convoluted. Could you please extract this check into a function and combine 1-2 `if` statements and `3-4` together?

File: src/main/views/video-hearings.njk [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      {{ govukErrorSummary({
+  "titleText": "There is a problem",
```
Comment (tautvydas-cerniauskas) [inline src/main/views/video-hearings.njk:20]: This text should be inside of a common translation file.

**Review by marcjudgey — APPROVED** [review]

**Review by tautvydas-cerniauskas — APPROVED** [review]

**Review by jsinghm13 — APPROVED** [review]
Approving on behalf of Marc

**Pull request merged**